subprojects/docs/src/snippets/kotlinDsl/assignment/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    `base`
+}
+
+// tag::assignment[]
+interface MyExtension {
+    val input: Property<String>
+    val output: RegularFileProperty
+}
```

Comment:
:x: Using `input` and `output` here is wrong. An extension defining "inputs" and "outputs" is strange, and using the same names for the properties in the extension and the task is also confusing. We could use e.g. `param1` and `param2` here instead.

Reply:
Removed and changed example.

Reply:
The new example is so much better and more relatable!